product_docs/docs/pgd/5/repsets.mdx

```diff
@@ -249,29 +249,29 @@ You can use [`bdr.replication_set_add_ddl_filter`](/pgd/latest/reference/repsets
 They're considered to be `DDL` and are therefore subject to DDL
 replication and global locking.
 
-## Selective Replication Example
+## Selective replication example
 
-In this example, we configure EDB Postgres Distributed to selectively replicate tables to particular groups of nodes. 
+This example configures EDB Postgres Distributed to selectively replicate tables to particular groups of nodes. 
 
-### Cluster Configuration
+### Cluster configuration
 
-This example assumes we have a cluster of six data nodes, `data-a1` to `data-a3` and `data-b1` to `data-b3` in two locations, represented by them being members of the `region_a` and `region_b` groups. 
+This example assumes you have a cluster of six data nodes, `data-a1` to `data-a3` and `data-b1` to `data-b3` in two locations. The two locations they're members of are represented as `region_a` and `region_b` groups. 
 
-There's also, as we recommend, a witness node, named `witness` in `region-c`, but we won't be needing to mention that in this example. The cluster itself will be called `sere`.
+There's also, as we recommend, a witness node named `witness` in `region-c` that isn't mentioned in this example. The cluster is called `sere`.
 
 This configuration looks like this:
 
 ![Multi-Region 3 Nodes Configuration](./planning/images/always-on-2x3-aa-updated.png)
 
-This is the standard Always-on multiregion configuration as discussed in the [Choosing your architecture](planning/architectures) section.
+This is the standard Always-on Multi-location configuration discussed in [Choosing your architecture](planning/architectures).
```

Comment:
I looked in the architecture doc and only saw multi-location, not multi-region. Please verify.

Reply:
Same thing. This is from when we used location and region interchangably. Use multi-region as the architecture docs will be being redone.